src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrise = 40;
+  const discontSevenDay = 50;
+  const discontThreeDay = 20;
```

Comment:
The variable names `basePrise` and `discontSevenDay` contain typos. According to the checklist, make sure that names are descriptive and correct. Consider renaming `basePrise` to `basePrice` and `discontSevenDay` to `discountSevenDays` to reflect their purposes accurately.